app/[team]/editor/article/edit/page.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+import { useParams } from 'next/navigation';
+import styled from 'styled-components';
+
+import AuthRequired from '@/components/auth/AuthRequired';
+import TextEditorBuild from '@/components/editor/TextEditorImport';
+import EditorInputTitle from '@/components/editor/ui/EditorInputTitle';
+import { useGetUpdateArticleContent } from '@/hooks/editor';
+import { TextEditorStyle } from '@/styles/TextEditorStyle';
+
+const EditArticlePage = () => {
+  const { articleId } = useParams();
+  const updateEditContents = useGetUpdateArticleContent(Number(78)); // number 값 articleId로 바꿀거이
+  // console.log(updateEditContents.data.content);
+  return (
+    <AuthRequired>
+      <TextEditorStyle>
+        <ArticleWrapper className="ProseMirror">
+          {/* 데이터가 content 있는 페이지 */}
+          {updateEditContents && (
+            <EditorInputTitle pageType="article" currentState="edit" data={updateEditContents.data} />
+          )}
+          {updateEditContents && (
+            <TextEditorBuild pageType="article" currentState="edit" data={updateEditContents.data} />
```

Comment:
```suggestion
          {updateEditContents && (
            <EditorInputTitle pageType="article" currentState="edit" data={updateEditContents} />
          )}
          {updateEditContents && (
            <TextEditorBuild pageType="article" currentState="edit" data={updateEditContents} />
```
이렇게 좀 더 깔끔하게 쓸 수 있을 것 같네요! data에 한번 더 접근 없이 미리 구조분해 할당으로 받아오니까! (그러나 로직을 자세히 살펴봐야 알겠지만 받아오는 데이터의 객체 구조에 대한 타입 설정이 잘 되어있지 않다면 타입 오류가 생길수도..!)